src/Microsoft.AspNetCore.CookiePolicy/ResponseCookiesWrapper.cs

```diff
@@ -10,7 +10,7 @@ namespace Microsoft.AspNetCore.CookiePolicy
 {
     internal class ResponseCookiesWrapper : IResponseCookies, ITrackingConsentFeature
```

Comment:
The class is internal so this won't help much

Reply:
How did I miss that, should I add a Defaults class?

Reply:
Wait, Damian gave me some additional feedback yesterday that may make this irrelevant.